src/lib.rs

```diff
@@ -81,10 +81,19 @@ unsafe extern "C" fn commit(
     txn: *mut pg::ReorderBufferTXN,
     _lsn: pg::XLogRecPtr,
 ) {
-    let s = CString::new("{ \"commit\": %u, \"t\": \"%s\" }").unwrap();
+    let j = r#"{"epoch": %u, "c_txid": %u, "commit": %u, "t": "%s", "xmin": %u, "c_xmin": %u}"#;
+    let s = CString::new(j).unwrap();
```

Comment:
This is a nitpick, and not required, but it'd be nice to change this to expect: `CString::new(j).expect("Failed to allocate CString");`